[sdl3-mixer] build error on x64-mingw-dynamic

Package: sdl3-mixer:x64-mingw-dynamic@3.2.0#1

**Host Environment**

- Host: x64-windows

- Compiler: GNU 15.2.0

- CMake Version: 4.3.1

- vcpkg-tool version: 2026-04-08-e0612b42ce44e55a0e630f2ee9d3c533a63d8bc1
  vcpkg-scripts version: 0b88aacde4 2026-04-11 (17 hours ago)

  **To Reproduce**

  `vcpkg install sdl3-mixer`

  **Failure logs**

```
-- Using cached libsdl-org-SDL_mixer-release-3.2.0.tar.gz
-- Cleaning sources at D:/vcpkg/buildtrees/sdl3-mixer/src/ease-3.2.0-36575875f8.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source D:/vcpkg/downloads/libsdl-org-SDL_mixer-release-3.2.0.tar.gz
-- Using source at D:/vcpkg/buildtrees/sdl3-mixer/src/ease-3.2.0-36575875f8.clean
-- Configuring x64-mingw-dynamic
-- Building x64-mingw-dynamic-dbg
-- Building x64-mingw-dynamic-rel
CMake Error at installed/x64-mingw-dynamic/share/vcpkg-cmake-config/vcpkg_cmake_config_fixup.cmake:36 (message):
  'D:/vcpkg/packages/sdl3-mixer_x64-mingw-dynamic/debug/cmake' does not
  exist.
Call Stack (most recent call first):
  ports/sdl3-mixer/portfile.cmake:53 (vcpkg_cmake_config_fixup)
  scripts/ports.cmake:206 (include)



```

--- a/ports/sdl3-mixer/portfile.cmake
+++ b/ports/sdl3-mixer/portfile.cmake
@@ -49,7 +49,7 @@ vcpkg_cmake_configure(
 
 vcpkg_cmake_install()
 vcpkg_copy_pdbs()
-if(VCPKG_TARGET_IS_WINDOWS)
+if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
     vcpkg_cmake_config_fixup(PACKAGE_NAME "SDL3_mixer" CONFIG_PATH "cmake")
 else()
     vcpkg_cmake_config_fixup(PACKAGE_NAME "SDL3_mixer" CONFIG_PATH "lib/cmake/SDL3_mixer")
@@ -58,6 +58,7 @@ vcpkg_fixup_pkgconfig()
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/share/licenses")
 
 file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/usage" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
 vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE.txt")

--- a/ports/sdl3-mixer/usage
+++ b/ports/sdl3-mixer/usage
@@ -1,4 +1,4 @@
 sdl3-mixer provides CMake targets:
 
-    find_package(SDL3_mixer CONFIG REQUIRED)
-    target_link_libraries(main PRIVATE $<IF:$<TARGET_EXISTS:SDL3_mixer::SDL3_mixer>,SDL3_mixer::SDL3_mixer,SDL3_mixer::SDL3_mixer-static>)
+  find_package(SDL3_mixer CONFIG REQUIRED)
+  target_link_libraries(main PRIVATE $<IF:$<TARGET_EXISTS:SDL3_mixer::SDL3_mixer>,SDL3_mixer::SDL3_mixer,SDL3_mixer::SDL3_mixer-static>)

--- a/ports/sdl3-mixer/vcpkg.json
+++ b/ports/sdl3-mixer/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "sdl3-mixer",
   "version": "3.2.0",
-  "port-version": 1,
+  "port-version": 2,
   "description": "An audio mixer that supports various file formats for Simple Directmedia Layer.",
   "homepage": "https://github.com/libsdl-org/SDL_mixer",
   "license": "Zlib",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3796,10 +3796,10 @@
       "baseline": "2.11.1",
       "port-version": 0
     },
-	"hical61-hical": {
-	  "baseline": "1.0.1",
-	  "port-version": 0
-	},
+    "hical61-hical": {
+      "baseline": "1.0.1",
+      "port-version": 0
+    },
     "hidapi": {
       "baseline": "0.15.0",
       "port-version": 1
@@ -9078,7 +9078,7 @@
     },
     "sdl3-mixer": {
       "baseline": "3.2.0",
-      "port-version": 1
+      "port-version": 2
     },
     "sdl3-shadercross": {
       "baseline": "3.0.0-preview2",

--- a/versions/s-/sdl3-mixer.json
+++ b/versions/s-/sdl3-mixer.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "ecd00493098fe345f1715c63e49d5aaaee8dc808",
+      "version": "3.2.0",
+      "port-version": 2
+    },
+    {
       "git-tree": "d971a06612eb93102cedb5189e0f32c050f80163",
       "version": "3.2.0",
       "port-version": 1

